tests/qos/test_qos_sai.py

```diff
@@ -142,6 +142,19 @@ def testQosSaiPfcXoffLimit(
         if "packet_size" in qosConfig[xoffProfile].keys():
             testParams["packet_size"] = qosConfig[xoffProfile]["packet_size"]
 
+        if "dualtor" in tbinfo["topo"]["name"]:
+            testParams["is_dualtor"] = True
+            if tbinfo["topo"]["type"] == 't0':
+                testParams["is_t0"] = True
```

Comment:
why is this check needed? if the dut is a dual ToR, then it implies it is a T0 topology

Reply:
will remove this duplicated check

Reply:
solved